### ISSUE TITLE
🎉Let's try v5 style prototype!

### ISSUE THREAD

**Issue by ElderJames** [ant-design-blazor/ant-design-blazor#4146]
Hello everyone , we are very happy that the early version for Ant Design v5 have been release to Nuget, and you can try it by install the package:

.csproj:
```xml
<PackageReference Include="AntDesign" Version="0.0.1-v5-*" />
```

_import.razor:
```cs
@using CssInCSharp
```

App.razor:
```razor
    <link no-antblazor-css />
    <StyleOutlet @rendermode="InteractiveServer" />
    @* <HeadOutlet @rendermode="InteractiveServer" /> *@
```

Please try and give us your feedback. We will continue the improvment for the development experience.

_Originally posted by @ElderJames in https://github.com/ant-design-blazor/ant-design-blazor/issues/2870#issuecomment-2239527335_

**Comment by ldsenow** [comment]
i will give it a go sometime this week. Thanks for the hard work!

**Comment by ElderJames** [comment]
Thank you @ldsenow , you helped a lot in this achievement!

**Comment by ldsenow** [comment]
I have applied the preview package in one of our apps. Overall works perfectly, great job. I found some minor issues.

- Tag's color is not applied
- Drowdown menu's border is too light if the background is also white
![image](https://github.com/user-attachments/assets/5e107f99-3b4c-41cb-a816-b7b46ea2f48d)
- Table header becomes a bit taller if there is sorting enabled
![image](https://github.com/user-attachments/assets/9cd4623f-5081-437e-b647-1ae75a1e3d12)

**Comment by ElderJames** [comment]
@ldsenow That's great! We expect you to bring more improvements to the project.

**Comment by IsaacPoon** [comment]
@ElderJames which version of AntDesign is this 0.0.5-v* bases on?

**Comment by ElderJames** [comment]
Hello @IsaacPoon , the 0.0.1-v5-* packages is base on antd v5.

**Comment by pathartl** [comment]
I haven't attempted to use this in my project yet, but I wanted to ask if there was a checklist of porting that needed to be completed. Having a clear list of changes that need to be made or components that need to built out would help for any prospective contributors.

**Comment by Seanxwy** [comment]
Is there any new development?

**Comment by ldsenow** [comment]
antd v6 is released. i guess we should give up v5 and css-in-cs. Consider the the native css variables and move onto v6 directly

**Comment by Seanxwy** [comment]
> antd v6 is released. i guess we should give up v5 and css-in-cs. Consider the the native css variables and move onto v6 directly

A very good proposal, but the current progress seems quite unsatisfactory. The custom theme function in v4 also has many flaws, with the problem of insufficient coverage. I don't know when it will be available.

**Comment by ElderJames** [comment]
@ldsenow Exactly, are you interested in helping to implement it?

**Comment by ldsenow** [comment]
> [@ldsenow](https://github.com/ldsenow) Exactly, are you interested in helping to implement it?

i am happy to help if you can give me a direction. Obviously not the full scope but love to involve. @ElderJames

**Comment by ElderJames** [comment]
Thank you @ldsenow , but I haven't found out the best way to achieve it.

**Comment by jstafford5380** [comment]
> Thank you [@ldsenow](https://github.com/ldsenow) , but I haven't found out the best way to achieve it.

I mean, in theory:

- https://digitsbits.com/what-are-design-tokens-css-and-how-to-use-them/
- https://www.designtokens.org/tr/drafts/format/

**Comment by ElderJames** [comment]
@jstafford5380 Yes, we still need to implement the Design Tokens for the style of components.

**Comment by Seanxwy** [comment]
> > 谢谢[@ldsenow](https://github.com/ldsenow)，但我还没找到实现它的最佳方法。
> 
> 我的意思是，理论上：
> 
> * https://digitsbits.com/what-are-design-tokens-css-and-how-to-use-them/
> * https://www.designtokens.org/tr/drafts/format/

Very good. I hope you can help make it happen.
